receiver/riakreceiver/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+name: riakreceiver
+
+attributes:
+  riak.node.name:
+    description: The name of the Riak node.
+  operation:
+    value: state
```

Comment:
This doesn't seem to match.

Reply:
This is still an issue.

`value: state` means that `state: put` is what appears in the end result. Get and Put aren't states, so this seems like a misnomer.

The `value` field is useful when you need to overwrite the name used above (`request.operation` in this case), but otherwise can be omitted.